src/scriptlets/prevent-refresh.js

```diff
@@ -84,11 +87,11 @@ export function preventRefresh(source, delaySec) {
         }
         let secondsToRun = getNumberFromString(delaySec);
         // Check if argument is provided
-        if (!secondsToRun) {
+        if (secondsToRun == null) {
```

Comment:
Strict equality `===` should be used instead of just `==`. and if you need to check whether the `secondsToRun` is defined, the condition can be updated to 
```
if (typeof secondsToRun !== 'undefined') { ... }
```

Reply:
Done.
I didn't use strict equality, because I thought that there may be situation where `secondsToRun` can be `undefined` and in another case when it can be `null`, but it seems that I was wrong.